app/views/images/new.html.erb

```diff
@@ -8,7 +8,8 @@
   <p><%= f.file_field :photo %></p>
 
   <p><%= f.text_field :title, {:placeholder => "Image title", :tabindex => 2} %></p>
-  <p><%= f.text_field :notes, {:placeholder => "a short description"} %></p>
+  <p><%= f.text_field :notes, {:placeholder => translation('images.short_description')} %></p>
+
```

Comment:
Hi @ASamedWalker, can you remove the empty line on line 12? Thanks!

Reply:
Made the changes